src/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -55,6 +57,9 @@
   <PropertyGroup Condition="'$(Platform)' == 'arm'">
     <DefineConstants>ARM;$(DefineConstants)</DefineConstants>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Platform)' == 'armel'">
+    <DefineConstants>ARMEL;$(DefineConstants)</DefineConstants>
```

Comment:
Or maybe even rewrite armel to arm at the top like it is done for CoreCLR: https://github.com/dotnet/coreclr/blob/master/src/mscorlib/System.Private.CoreLib.csproj#L16